uwh/xbee_comms.py

```diff
@@ -0,0 +1,80 @@
+from digi.xbee.devices import XBeeDevice
+from digi.xbee.exception import TimeoutException
+
+from . import messages_pb2
+from .comms import UWHProtoHandler
+
+import time
+
+class XBeeClient(UWHProtoHandler):
+    def __init__(self, mgr):
+        UWHProtoHandler.__init__(self, mgr)
+        self._xbee = XBeeDevice('/dev/tty.usbserial-DN040E8Y', 9600)
```

Comment:
I plan on exposing the tty and baud rate as parameters soon, and they'll be stored in one of those config files (like the screen size stuff in uwh-refbox).